.github/workflows/codeql.yml

```diff
@@ -352,8 +352,8 @@ jobs:
         import shutil
         from pathlib import Path
 
-        # Only these two plugins are needed for CodeQL
-        plugins_to_keep = ['CodeQL', 'CompilerPlugin']
+        # Only this plugin is needed for CodeQL

```

Comment:
This doesn't seem to match the [current file](https://github.com/microsoft/mu_devops/blob/020324d18d15b7493265b0ce5860f9c0233876b5/.sync/workflows/leaf/codeql.yml#L355) being synced so it would be overridden:

```python
        # Only these two plugins are needed for CodeQL
        plugins_to_keep = ['CodeQL', 'CompilerPlugin']
```

Reply:
Let me check on this file. It should not be here, I think...

Reply:
Not sure how it creeped into the last commit of this series. But I removed it and will see how that works.

Reply:
I believe that was just a temporary work-around (at the time) when Ken was getting everything worked out with 2311. It's definitely not needed now - the sync'd file should work as expected :)